main.js

```diff
@@ -9,7 +9,7 @@ const createWindow = function createWindow() {
     title:      'To The End',
     width:      1600,
     height:     900,
-    fullscreen: true,
+    fullscreen: false,
```

Comment:
Let's not commit this. I'll set up a PR that disables fullscreen when running `npm start`.